_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -2044,11 +2043,6 @@ mfxStatus ImplementationAvc::QueryFromMctf(void *pParam, bool bEoF)
     if (IsCmNeededForSCD(m_video))
     {
         MFX_SAFE_CALL(amtMctf->MCTF_RELEASE_FRAME());
-        if (m_cmDevice && pTask->m_cmRawForMCTF)
-        {
-            m_cmDevice->DestroySurface(pTask->m_cmRawForMCTF);
```

Comment:
Where did you move destroying of the created surface?

Reply:
This has changed and will be handled now by QfIn

Reply:
This surface no longer exist, there is no creation or destruction for cmRawMCTF